src/components/PageMetadata/PageMetadata.tsx

```diff
@@ -1,33 +1,36 @@
-
 import React, { useEffect, useState } from 'react';
+import { Stack } from '@fluentui/react/lib/Stack';
```

Comment:
👍  Nice but what a special feature of this component can we just just flex wrap?

Reply:
Quick usage LOL